### ISSUE TITLE
Suggestion: Ability to move the taskbar to the top or the sides

### ISSUE THREAD

**Issue by santumerino** [ChrisAnd1998/TaskbarXI#8]
I have my taskbar at the top but on Windows 11 they're getting rid of the ability to move it. I'd be eternally grateful if you could fix what Microsoft broke 🙏🙏🙏

**Comment by Tsury** [comment]
Please make it happen

**Comment by seffyroff** [comment]
I have 4 monitors in a 2x2 arrangement.  Right now I have 2 taskbars running across the bottom of the top row, which is really annoying.  

On Win10 I put my taskbars to the left and right edges (at the extremities of the displays).  Win11 currently doesn't permit me to do this.  I've gone as far as rolling back to Win10 as the Win11 layout is completely unusable for me.

**Comment by maxfreytag** [comment]
Yes please! Left and right would be so awesome. On 21:9 bottom/top makes simply no sense...

**Comment by astroajay** [comment]
Keeping my dock on the left spouse of my screen has always been useful because it starts out of the way, Microsoft broke that, please add a feature to fix that, pretty please...

**Comment by jpetays** [comment]
On Win10 I put my taskbar to the left of my right most monitor so it is in convenient place to access on both monitors.
Further more as a programmer I need more vertical space (lines to see) than horizontal in my 16:10 aspect ratio monitors.
